data/json/terrain.json

```diff
@@ -8547,6 +8914,82 @@
       ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_chimney",
+    "name": "chimney crown",
+    "description": "The top of a chimney, it looks sooty.",
+    "symbol": "#",
+    "color": "red",
+    "move_cost": 2,
+    "roof": "t_flat_roof",
+    "flags": [ "BASHABLE", "MINEABLE" ],
+    "bash": {
+      "str_min": 12,
+      "str_max": 25,
+      "sound": "whack!",
+      "sound_fail": "whump!",
+      "ter_set": "t_null",
+      "items": [ { "item": "brick", "count": [ 3, 30 ] } ]
```

Comment:
Most chimneys also have a large metal tube in them, either a pipe or a sheet metal.